src/otx/core/utils/config.py

```diff
@@ -60,16 +60,6 @@ def convert_conf_to_mmconfig_dict(
     raise ValueError(to)
 
 
-def mmconfig_dict_to_dict(obj: MMConfig | list[MMConfig]) -> list | dict:
-    """Convert MMEngine config object to Python dictionary."""
-    if isinstance(obj, list):
-        return [mmconfig_dict_to_dict(x) for x in obj]
-    if hasattr(obj, "to_dict"):
-        return {k: mmconfig_dict_to_dict(v) for k, v in obj.to_dict().items()}
-
-    return obj
-
-
 def inplace_num_classes(
```

Comment:
Is this function used anywhere else?

Reply:
Thanks for catching this, removed at 44bff8d.